synth.py

```diff
@@ -26,11 +24,6 @@
 templates = common_templates.node_library()
 s.copy(templates, excludes=[".github/CONTRIBUTING.md", "README.md"])
 
-# repository_url = "https://github.com/googleapis/google-api-nodejs-client.git"
-# log.debug(f"Cloning {repository_url}.")
-# repository = git.clone(repository_url, depth=1)
-# log.debug("Installing dependencies.")
-# shell.run(["npm", "install"], cwd=repository)
-# log.debug("Generating all libraries...")
-# shell.run(["npm", "run", "generate"], cwd=repository)
-# s.copy(repository / "src")
+subprocess.run(["npm", "install"])
+subprocess.run(["npm", "run", "generate"])
```

Comment:
@bcoe one thing to look out for here was creating timeouts with autosynth. Would be good to keep an eye if we do in fact decide to do this.